test/sequential/test-util-deep-equal.js

```diff
@@ -0,0 +1,27 @@
+var assert = require('assert');
```

Comment:
is there a reason this and the following test are in `sequential` rather than `parallel`? usually tests go into the latter unless they have finicky async stuff